snappy-tools/src/main/scala/io/snappydata/ToolsCallbackImpl.scala

```diff
@@ -37,16 +37,21 @@ object ToolsCallbackImpl extends ToolsCallback {
   override def invokeStartFabricServer(sc: SparkContext,
       hostData: Boolean): Unit = {
     val properties = new Properties()
-    sc.getConf.getOption(Property.locators).map { locator =>
-      if (!Utils.LocatorURLPattern.matcher(locator).matches()) {
-        throw new AnalysisException(s"locators property '$locator' should " +
-            "be provided in the format host[port] or host:port", null)
-      }
-      properties.setProperty("locators", locator)
-      sc.getConf.getOption(Property.mcastPort).map(
-        properties.setProperty("mcast-port", _))
-    }.getOrElse(properties.setProperty("mcast-port",
-      sc.getConf.get(Property.mcastPort)))
+    sc.getConf.getAll.filter(
+      p => p._1.startsWith(Constant.STORE_PROPERTY_PREFIX)).
+        foreach(p => p._1 match {
```

Comment:
okay they are different blocks. But changing name would still be good from reading perspective.